kolibri/core/assets/src/vue/content-renderer/index.vue

```diff
@@ -174,9 +178,26 @@
           Object.assign(options, this.currentViewClass);
           // Instantiate the Vue instance directly using the Kolibri Vue constructor.
           this.contentView = new this.Kolibri.lib.vue(options); // eslint-disable-line new-cap
+          this.contentView.$on('startTracking', this.startTracking);
+          this.contentView.$on('stopTracking', this.stopTracking);
+          this.contentView.$on('progressUpdate', this.updateProgress);
+          this.initSession(this.Kolibri, this.channelId, this.contentId, this.kind);
         }
       },
     },
+    vuex: {
+      actions: {
+        initSession: actions.initContentSession,
+        updateProgress: actions.updateProgress,
+        startTracking: actions.startTrackingProgress,
+        stopTracking: actions.stopTrackingProgress,
+        saveLogs: actions.saveLogs,
```

Comment:
`stopTracking` seems to get fired as an event, but don't see where the event is getting picked up and triggering the action.